File: Basic_algorithm/1_sum_of_two_digits/sum_two_number.c
#include<stdio.h>
int sum_of_total_number(int first, int second){
    return first + second; //Do sum;
}

int main(){

    int a=0,b=0;
    printf(" Input a and b in digits: \'use space\' ");
    scanf("%d%d",&a,&b);
    printf(" Sum of total number \t %d\n",sum_of_total_number(a,b));
    return 0;
}

File: Basic_algorithm/1_sum_of_two_digits/sum_two_number.r
sum_of_total_number <- function (first,second){
  first <-  as.numeric(first)
  second <-  as.numeric(second)
  return (first + second )
}

main <- function(a,b){
  { a <- readline("Input value a: "); b <- readline("INput value b: ")}
  print(sum_of_total_number(a,b))
}

main()
